app/models/submission.rb

```diff
@@ -0,0 +1,40 @@
+class Submission < ApplicationRecord
+  belongs_to :user
```

Comment:
thoughts on referring to this relationship as an `owner`, `submitter`, or a `poster`?

Reply:
If its possible to set up 2 kinds of roles with scope, I'd like `post_submitter` and `commenter`. I guess `poster` and `commenter` could better reinforce the objects structure. If setting up 2 role types is difficult, then I'm up for `submitter` in place of `user`.

Reply:
Hmmm if you wanted, an `alias` *could* be setup to have one term alias the `belongs_to` relationship, but since this is a more generic submission model, I would go with a more generic name for the submitter.

Reply:
On second thought, `alias` would not work in this instance (sorry, it's been a while since I've Rails'd). The only option to make this work here would be setting 2 `belongs_to` statements. Having 1 sounds better to me.

Other option is to split this model into 2: `Post` and `Comment`. That would allow you to be more specific in roles and functionality.

Reply:
Hmm... I want to stick with keeping posts and comments in the same model for now and see if I can set up the data table to accommodate nested comments. If that gets too cumbersome, I'll split the model. At the moment, I don't think its super crucial to make a distinction between poster and commenter.

Reply:
I'm trying to move forward with changing this to `:submitter`, but I have more questions

First - syntax-wise - should the following work?
`belongs_to :submitter, class_name: "User"`

Second, I tried using the syntax above and was getting errors when I tried to rake the seeds file because 'user' wasn't recognized, which I’m guessing is because some of my validations will need to be updated from `user` to `submitter`.

Third, were you thinking that `submitter` would completely replace `user` or be in addition to `user`

Reply:
> First - syntax-wise - should the following work?

I think you might need to specific the `foreign_key` as Rails will assume `submitter_id`.  If this is going to be too much of a hassle, then we can pass on doing this. I just thought it'd be nice to have it be a bit more specific as to the role the user was on this table.

> Second...

Might be that, but could also be the foreign key thing that I mentioned above.

> Third, were you thinking that submitter would completely replace user or be in addition to user

The former -- completely replace `user` here with `submitter`.